numpy/random/generator.pyx

```diff
@@ -920,7 +920,7 @@ cdef class Generator:
         """
         key = np.dtype(dtype).name
         if key == 'float64':
-            return double_fill(&random_gauss_zig_fill, &self._bitgen, size, self.lock, out)
+            return double_fill(&random_gauss_zig, &self._bitgen, size, self.lock, out)
```

Comment:
I **think** you can go further if the fill-methods are being removed and you can remove double_fill and just use 

```
cont(void *random_gauss_zig, &self._bitgen, size, self.lock, 0,
     0.0, '', CONS_NONE, None,0.0, '', CONS_NONE, None, 0.0, '', CONS_NONE, None
     out)
```
```